app/lib/index.js

```diff
@@ -756,10 +754,18 @@ function bootstrap() {
   // (9) zeebe API
   const zeebeAPI = new ZeebeAPI({ readFile }, Camunda8, flags);
 
-  // (10) connector templates
-  if (!connectorTemplatesDisabled) {
-    new TemplatesUpdater(renderer, config, userPath);
-  }
+  // (10) template updater
+  const templateUpdater = new TemplateUpdater(config, {
+    defaultEndpoints: [ OOTB_CONNECTORS_ENDPOINT ]
+  });
+
+  templateUpdater.onUpdated(event => {
+    renderer.send('client:templates-update-success', event.hasNew, event.warnings);
+  });
+
+  renderer.on('client:templates-update', ({ executionPlatform, executionPlatformVersion }) => {
+    templateUpdater.updateTemplates(executionPlatform, executionPlatformVersion);
```

Comment:
This will always fetch the templates regardless of whether they are enabled or not.